openslides/__main__.py

```diff
@@ -104,9 +104,9 @@ def get_parser():
         help='The used settings file. The file is created, if it does not exist.')
     subcommand_start.set_defaults(callback=start)
     subcommand_start.add_argument(
-        '--development',
+        '--local-installation',
         action='store_true',
-        help='Option for development purposes.')
+        help='Store settings and user files in a local directory.')
```

Comment:
No: You have to use this option if you want to start OpenSlides and use the settings over there. Else OpenSlides tries to find the settings deep inside the user path.